src/pgrn-compatible.h

```diff
@@ -191,6 +191,16 @@
 #	define PGRN_INDEX_AM_ROUTINE_HAVE_AM_PARALLEL_VACUUM_OPTIONS
 #endif
 
+#if PG_VERSION_NUM >= 150000
+#   define PGRN_RELKIND_HAS_TABLE_AM(relkind) \
+	RELKIND_HAS_TABLE_AM(relkind)
+#else
+#   define PGRN_RELKIND_HAS_TABLE_AM(relkind) \
+	((relkind) == RELKIND_RELATION || \
+	 (relkind) == RELKIND_TOASTVALUE || \
+	 (relkind) == RELKIND_MATVIEW)
+#endif
```

Comment:
Could you unify this to `#if PG_VERSION_NUM >= 150000` for `PGRN_RELKIND_HAS_PARTITIONS`?

FYI: You can use region and `\C-c \C-\` to write/format `\` for continuous lines in Emacs.